test/makeup/lexers/elixir_lexer/elixir_lexer_tokenizer_test.exs

```diff
@@ -50,6 +50,37 @@ defmodule ElixirLexerTokenizerTestSnippet do
       assert lex("...(12)> ") == [{:generic_prompt, %{selectable: false}, "...(12)> "}]
     end
 
+    test "parses multiline iex prompt correctly" do
+      source = """
+      iex> x = [
+      ...>   key:
```

Comment:
Previously the `keyword` tokenizer would consume `key:` and also the newline. However, the newline is important for the iex prompt to match:

https://github.com/elixir-makeup/makeup_elixir/blob/899866c8a2e97b6a6c11f1722d1d842092367362/lib/makeup/lexers/elixir_lexer.ex#L311-L313

As a result the above wouldn't match and `...` would be tokenized as `:name`.